web/packages/api/src/status.ts

```diff
@@ -65,15 +65,17 @@ export enum AlarmReason {
     AccountBalanceInsufficient = "AccountBalanceInsufficient",
     ToEthereumNoTransfer = "ToEthereumNoTransfer",
     ToPolkadotNoTransfer = "ToPolkadotNoTransfer",
+    ToEthereumChannelAttacked = "ToEthereumChannelAttacked",
+    ToPolkadotChannelAttacked = "ToPolkadotChannelAttacked"
 }
 
 export type Sovereign = { name: string; account: string; balance: bigint; type: SourceType }
 
 export const BlockLatencyThreshold = {
-    // Syncing beefy finality update every 4 hours(2400 blocks) so we set 3000 blocks at most.
-    ToEthereum: 3000,
-    // Syncing beacon finality update every 6.4 minutes(32 blocks) so we set 128 blocks (4 epochs) at most.
-    ToPolkadot: 128,
+    // Syncing beefy finality update every 4 hours(1200 ethereum blocks)
+    ToEthereum: 1200,
```

Comment:
Is this the change that will improve the toEthereum false positive alarm? Can you please explain why you think the false alarm happens? :)

Reply:
Also, remember to include 35 minutes for the interactive protocol.

Reply:
> Is this the change that will improve the toEthereum false positive alarm?

Nope, it's because `SCAN_INTERVAL` we config is less than the alarm period and config `TreatMissingData = "breaching"`